src/components/Request/NewRequestForm.tsx

```diff
@@ -139,23 +203,79 @@ const NewRequestForm = () => {
 
         <Grade name="grade" form={myForm} />
 
+        <OfficeSymbol name="officeSymbol" form={myForm} />
+
+        <Supervisor name="supervisor" form={myForm} />
```

Comment:
Card says this field should provide Rank, but I don't think we have a way to get that via SP Person Profile